.jenkins/actions/run_performance.sh

```diff
@@ -6,11 +6,14 @@ ROOT_DIR="$(dirname "$(dirname "$SCRIPTPATH")")"
 test -n "${experiment}" || exitError 1001 ${LINENO} "experiment is not defined"
 test -n "${backend}" || exitError 1002 ${LINENO} "backend is not defined"
 
+if [ "$experiment" = "c12_6ranks_standard" ]; then
+    data_path="/project/s1053/fv3core_serialized_test_data/7.0.0/c12_6ranks_standard"
+fi
 if [ "$experiment" = "c96_6ranks_baroclinic" ]; then
     data_path="/project/s1053/fv3core_serialized_test_data/7.2.3/c96_6ranks_baroclinic"
 fi
-if [ "$experiment" = "c12_6ranks_standard" ]; then
-    data_path="/project/s1053/fv3core_serialized_test_data/7.0.0/c12_6ranks_standard"
+if [ "$experiment" = "c128_6ranks_baroclinic" ]; then
+    data_path="/project/s1053/fv3core_serialized_test_data/7.2.3/c128_6ranks_baroclinic"
 fi
 
-$ROOT_DIR/examples/standalone/benchmarks/run_on_daint.sh 60 6 $backend /project/s1053/performance/fv3core_monitor/$backend/ $data_path
+$ROOT_DIR/examples/standalone/benchmarks/run_on_daint.sh 2 6 $backend /project/s1053/performance/fv3core_monitor/$backend/ $data_path
```

Comment:
how long do 2 timesteps take with c128?

Reply:
my last run was this:
```
{
    "setup": {
        "data set": "c128_6ranks_baroclinic",
        "experiment time": "03/02/2021 06:05:19",
        "hash": "38c6b175feb24bc2b6ab85b40ea3660729a60238",
        "timesteps": 2,
        "version": "python/gtx86"
    },
    "times": {
        "cleanup": {},
        "init": {
            "maximum": 5451.73504781723,
            "mean": 5451.671283125877,
            "median": 5451.678248047829,
            "minimum": 5451.607561826706
        },
        "main_loop": {
            "maximum": 3374.746320962906,
            "mean": 3374.6829599936805,
            "median": 3374.6760544776917,
            "minimum": 3374.619304895401
        },
        "total": {
            "maximum": 8826.354474067688,
            "mean": 8826.354243119558,
            "median": 8826.35430252552,
            "minimum": 8826.353882789612
        }
    }
}
```
so somewhere between 2 and 3 hours